osu.Game.Tournament/Components/TournamentMatchChatDisplay.cs

```diff
@@ -66,6 +66,10 @@ private void load(MatchIPCInfo ipc)
             }
         }
 
+        public void Expand() => this.FadeIn(300);
+
+        public void Contract() => this.FadeOut(200);
```

Comment:
Was it intended that the Y movement logic was culled? After this PR it seems to look broken:

![image](https://user-images.githubusercontent.com/20418176/76164943-3fdf4c80-6153-11ea-915f-81c38b6ce75a.png)

(screenshot taken in `TestSceneGameplayScreen` after jerry-rigging it to have a channel with messages).

Reply:
it gets fixed in the gameplay screen update. i just removed here for simplicity.

Reply:
Figured that might be the case, but it was going to be a while until I got to it. Good to know anyways 👍